lib/src/Dropdown/Dropdown.tsx

```diff
@@ -176,22 +176,22 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(({ wrapperProp
         return list?.length ? list : searchKeyword ? <p>{text.noResult || defaultText.noResult}</p> : <p>{text.emptyList || defaultText.emptyList}</p>;
     };
 
-    if (!isMobile) {
-        React.useEffect(() => {
-            props.multiple && setAllSelected(Array.from(selectRef.current.options).every((option) => option.selected));
-        }, [props.value]);
+    React.useEffect(() => {
+        !isMobile && props.multiple && setAllSelected(Array.from(selectRef.current.options).every((option) => option.selected));
+    }, [props.value]);
```

Comment:
Actually not sure if it's supposed to be props.value here as well, but you missed out props.multiple

Reply:
No, it's not supposed to be `props.multiple`. This is an effect triggered by the change of value, but only applied when it's of type `multiple`.

Reply:
Ok if that's the case you need to add both value and multiple to the deps array. Because you might have the prop multiple which would change between renders but not the value in which case you would still want this effect to run correct?